include/nlohmann/detail/iterators/iteration_proxy.hpp

```diff
@@ -6,6 +6,10 @@
 #include <tuple> // tuple_size, get, tuple_element
 #include <utility> // move
 
+#if JSON_HAS_RANGES
+    #include <ranges> // enable_borrowed_range
```

Comment:
I think it would be a good idea to add some examples for using ranges to the API page of `items`. What do you think?

Reply:
There're a few place where examples would be helpful. I was being lazy. ;-)

But ranges are tricky anyway because of the mixed and at times sparse support in the big standard library implementations. The unit test is similarly meager because testing and guarding each algorithm/view is time consuming.
Then there's the incompatibility between the GNU standard library and Clang.

Reply:
I totally understand! Then let's not overwhelm this PR and have the examples as homework for later...